src/components/app.js

```diff
@@ -2,10 +2,21 @@
 import React from 'react';
 import BooksList from './bookslist';
 import BooksForm from './book-form';
+import Header from './header';
 
+const style = {
```

Comment:
When the book list is not filled, the whole thing shifts down, leaving some space at the top.

![Screenshot 2019-12-04 at 10 01 32 AM](https://user-images.githubusercontent.com/10922508/70128490-35f20d00-167d-11ea-8a52-3a9a5d530054.png)